incubator/drone/values.yaml

```diff
@@ -0,0 +1,41 @@
+## Drone
+## ref: https://hub.docker.com/r/drone/drone
+##
+image:
+  registry: docker.io
+  org: drone
+  name: drone
+  tag: 0.5
+  pullPolicy: IfNotPresent
+
+service:
+  http:
+    externalPort: 80
+    internalPort: 8000
+  type: ClusterIP
+
+server:
+  # Drone server configuration. Values in here get injected as environment variables.
+  # See http://readme.drone.io/admin/installation-reference#server-options for a list of possible values.
+  env:
+    DRONE_DEBUG: "false"
```

Comment:
This section should also include some commented out environment variables for git service setup, or at least a note that without a git service configured drone-server will fail to start.